From e8eb59c93abd7e835cb5a1488a4679b585985715 Mon Sep 17 00:00:00 2001
From: Adam Sampson <ats@offog.org>
Date: Sun, 5 Nov 2017 11:45:00 +0000
Subject: [PATCH] Fix incorrect paths in .pc files.

The change in 67c3878ef10449b241c37d1022e2518029860335 used the
CMAKE_INSTALL_x variables instead of CMAKE_INSTALL_FULL_x; the former
contain things like "include" instead of full paths, so the resulting
.pc files break packages that depend on poppler. Use the latter.

Also fix a missing @ in the qt5 file.

Bug #103578
---
 poppler-cairo.pc.cmake  | 4 ++--
 poppler-cpp.pc.cmake    | 4 ++--
 poppler-glib.pc.cmake   | 4 ++--
 poppler-qt4.pc.cmake    | 4 ++--
 poppler-qt5.pc.cmake    | 4 ++--
 poppler-splash.pc.cmake | 4 ++--
 poppler.pc.cmake        | 4 ++--
 7 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/poppler-cairo.pc.cmake b/poppler-cairo.pc.cmake
index 4a606440..e68cceab 100644
--- a/poppler-cairo.pc.cmake
+++ b/poppler-cairo.pc.cmake
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@CMAKE_INSTALL_LIBDIR@
-includedir=@CMAKE_INSTALL_INCLUDEDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
 Name: poppler-cairo
 Description: Cairo backend for Poppler PDF rendering library
diff --git a/poppler-cpp.pc.cmake b/poppler-cpp.pc.cmake
index a26177a5..3eb68b38 100644
--- a/poppler-cpp.pc.cmake
+++ b/poppler-cpp.pc.cmake
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@CMAKE_INSTALL_LIBDIR@
-includedir=@CMAKE_INSTALL_INCLUDEDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
 Name: poppler-cpp
 Description: cpp backend for Poppler PDF rendering library
diff --git a/poppler-glib.pc.cmake b/poppler-glib.pc.cmake
index 6e006d49..ac248192 100644
--- a/poppler-glib.pc.cmake
+++ b/poppler-glib.pc.cmake
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@CMAKE_INSTALL_LIBDIR@
-includedir=@CMAKE_INSTALL_INCLUDEDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
 Name: poppler-glib
 Description: GLib wrapper for poppler
diff --git a/poppler-qt4.pc.cmake b/poppler-qt4.pc.cmake
index 8523a17a..46a37f6d 100644
--- a/poppler-qt4.pc.cmake
+++ b/poppler-qt4.pc.cmake
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@CMAKE_INSTALL_LIBDIR@
-includedir=@CMAKE_INSTALL_INCLUDEDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
 Name: poppler-qt4
 Description: Qt4 bindings for poppler
diff --git a/poppler-qt5.pc.cmake b/poppler-qt5.pc.cmake
index aaa1aa77..94636892 100644
--- a/poppler-qt5.pc.cmake
+++ b/poppler-qt5.pc.cmake
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-libdir=CMAKE_INSTALL_LIBDIR@
-includedir=@CMAKE_INSTALL_INCLUDEDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
 Name: poppler-qt5
 Description: Qt5 bindings for poppler
diff --git a/poppler-splash.pc.cmake b/poppler-splash.pc.cmake
index 1aa1a835..f9eceebe 100644
--- a/poppler-splash.pc.cmake
+++ b/poppler-splash.pc.cmake
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@CMAKE_INSTALL_LIBDIR@
-includedir=@CMAKE_INSTALL_INCLUDEDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
 Name: poppler-splash
 Description: Splash backend for Poppler PDF rendering library
diff --git a/poppler.pc.cmake b/poppler.pc.cmake
index 9432c5c6..00b73488 100644
--- a/poppler.pc.cmake
+++ b/poppler.pc.cmake
@@ -1,6 +1,6 @@
 prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@CMAKE_INSTALL_LIBDIR@
-includedir=@CMAKE_INSTALL_INCLUDEDIR@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
 Name: poppler
 Description: PDF rendering library
-- 
2.15.0

